its/src/test/java/com/sonar/maven/it/suite/MavenTest.java

```diff
@@ -500,6 +500,20 @@ void skipIrrelevantEncryptedMavenProperties() {
     assertThat(buildResult.isSuccess()).isTrue();
   }
 
+  /**
+   * SCANMAVEN-228
+   */
+  @Test
+  void skipIrrelevantEncryptedMavenPom() {
```

Comment:
The test name might be a little confusing here since the whole POM is not encrypted. What about
```suggestion
  void skipIrrelevantEncryptedPropertyInMavenPom() {
```